### PR TITLE
populate user personal dashboard data on home page

### PR DESCRIPTION
**Pull request by petrjasek**
CPCN-249

### CODE REVIEW

File: newsroom/wire/views.py [hunk 1]
```diff
@@ -161,6 +193,7 @@ def get_home_data():
         "topics": topics,
         "ui_config": get_resource_service("ui_config").get_section_config("home"),
         "groups": app.config.get("WIRE_GROUPS", []),
+        "personalizedDashboards": get_personal_dashboards_data(user, company, topics),
```
Comment (MarkLark86) [inline newsroom/wire/views.py:196]: Loading the items on initial load goes against the responsive changes we did previously. At the moment we leave it to the client to request these items, and show a loading spinner until the items arrive on the client.

File: newsroom/search/service.py [hunk 2]
```diff
@@ -866,3 +825,61 @@ def get_matching_topics_for_item(self, topics, users, companies, query):
             raise
 
         return topic_matches
+
+    def get_topic_query(self, topic, user, company, section_filters, query=None):
```
Comment (MarkLark86) [inline newsroom/search/service.py:829]: I've done something similar with my work on scheduled notifications. And as this is in the typescript branch, we will have merge conflicts. In this case, would it be better to base/merge this into the `develop` branch first then cherry-pick into typescript branch.

This way I can rebase from latest develop merging your changes with mine

File: newsroom/wire/views.py [hunk 3]
```diff
@@ -134,6 +134,38 @@ def delete_dashboard_caches():
         app.cache.delete(f"{HOME_ITEMS_CACHE_KEY}{company['_id']}")
 
 
+def get_personal_dashboards_data(user, company, topics):
+    section_filters = get_resource_service("section_filters").get_section_filters_dict()
+
+    def get_topic_items(topic):
+        query = superdesk.get_resource_service("wire_search").get_topic_query(topic, user, company, section_filters)
```
Comment (MarkLark86) [inline newsroom/wire/views.py:141]: Should we also provide a `size` param as to limit the number of items returned for these homepage cards?

File: newsroom/wire/views.py (hunk 1 above)
Comment (MarkLark86) [reply newsroom/wire/views.py:196]: We are currently doing that loading in the dashboard though.

And I agree about caching, similar to what we have with current dashboard items

File: newsroom/search/service.py (hunk 2 above)
Comment (petrjasek) [reply newsroom/search/service.py:829]: ok np

File: newsroom/wire/views.py (hunk 3 above)
Comment (petrjasek) [reply newsroom/wire/views.py:141]: I have it set in the `get_items_by_query` method

**Pull request closed without merging**